pkgs/data/documentation/zeal/default.nix

```diff
@@ -22,10 +34,10 @@ mkDerivation rec {
       -e 's@^project.*@project(Zeal VERSION ${version})@'
   '';
 
-  nativeBuildInputs = [ cmake extra-cmake-modules pkg-config ];
+  nativeBuildInputs = [ cmake extra-cmake-modules pkg-config wrapQtAppsHook ];
 
   buildInputs = [
-    qtbase qtimageformats qtwebengine qtx11extras
```

Comment:
`qtx11extras` seems unneeded and is unavailable on qt6 however unsure if it is needed on other platforms.